### DISCUSSION TITLE
[Feature] Share albums without photo datetime information

### DISCUSSION THREAD

**Discussion by vanchaxy** [immich-app/immich#14942] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I want to be able to share albums as a set of photos without information on when the photos were taken. 

Photos can still have the same order just without a date/time. 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This has been supported for a while now
